src/consumer/consumer-locator.js

```diff
@@ -11,6 +11,10 @@ function composeBitHiddenDirPath(path: string) {
   return pathlib.join(path, BIT_HIDDEN_DIR);
 }
 
+function composeBitGitHiddenDirPath(path: string) {
+  return pathlib.join(path, 'git', 'bit');
```

Comment:
why aren't you use the constants? also isn't it should be .git and not git?

Reply:
my mistake forgot about that :(